src/Initializer/typedefs.hpp

```diff
@@ -460,16 +462,14 @@ struct BoundaryFaceInformation {
  * See, an example of a policy-based design in GlobalData.cpp
  * */
 struct MemoryProperties {
-  size_t alignment{ALIGNMENT};
-  size_t pagesizeHeap{PAGESIZE_HEAP};
-  size_t pagesizeStack{PAGESIZE_STACK};
+  size_t alignment{Alignment};
+  size_t pagesizeHeap{PagesizeHeap};
+  size_t pagesizeStack{PagesizeStack};
 };
```

Comment:
Why do we need this struct? If all of these quantities are `constexpr`, we can always refer to them directly, insted of using this struct, which could be changed.

Reply:
They are used in the `GlobalData` allocation only—the values are set according to us being on host or device. We may be good with either keeping everything as is—or we could be moving the `struct` just there to `src/Initializer/GlobalData.h`.

Reply:
Then, I'd move the struct.

Reply:
Done.